tools/templates/overview.mustache

```diff
@@ -203,16 +203,7 @@
        clientId: '{{config.oidc.clientId}}',
```

Comment:
In this overview template, remove the login-custom link (for now).  We'll bring this back when we revamp the samples to the V2 strategy.

Reply:
Done